src/core_modules/capture-core/events/mainConverters/mainEventClientToServerConverter.js

```diff
@@ -32,6 +32,9 @@ export function convertMainEventClientToServer(event: Object, serverMinorVersion
         case 'assignee':
             convertedValue = value && convertClientToServer(value, dataElementTypes.ASSIGNEE);
             break;
+        case 'orgUnit':
```

Comment:
This change breaks other functionality where the `convertMainEventClientToServer` is used. For example when adding an event in [getConvertedAddEvent](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/getConvertedAddEvent.js#L34) or when assigning a user in [viewEvent.selectors](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/ViewEvent/ViewEventComponent/viewEvent.selectors.js#L30). Could you have a look? Thank you!